src/Styles.jsx

```diff
@@ -27,11 +27,23 @@ export const RecItemTitle = styled.p`
 
 export const RecList = styled.ul`
   list-style-type: none;
-  display: inline-block;
+  display: flex;
+  flex-direction: horizontal;
+  justify-content: left;
+  :hover button {
+    visibility: visible;
+    opacity: 1;
+    transition: opacity 0s linear;
+  }
```

Comment:
so you are referencing const NavButton = styled.button, basically?

Reply:
basically. I'm saying, "on hover of RecList, any children who are of type button should have this style." In this case, there are two.